### DISCUSSION TITLE
Teachers benefits; copilot

### DISCUSSION THREAD

**Discussion by arifrilander** [community/community#120968] [GitHub Education]
### Select Topic Area

Question

### Body

While trying to get benefits for a teacher, I got following message. 

"The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket."

**The school DOES appear in Finland. It is even in your map. Check the attachment.** I have colleagues who have copilot for VSC installed and running. How can this school have incorrect or incomplete info, because it is working for my colleagues.
![IMG_2521](https://github.com/community/community/assets/167980222/be58209a-9c7b-4e39-9def-e80dffc5f142)

**Comment by Akash1134** [comment]
Welcome to the GitHub Community, @arifrilander , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
